README.md

```diff
@@ -1,24 +1,43 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+# Kamzzy Blog
+> A blog application with lots of funtionalities.
+
+ ***
+
+## Built With
+> Ruby
+***
+## Getting Started
+
+* Clone the repository to get a local copy
+* Run `bundle install` to install all depndencies 
+* Run the app using `rails s`
+***
+##  unit testing
+* Run tests using rspec 
+***
+### Prerequisites
+> Ruby
+***
+### Author
+
+👤 Onyeagoziri Precious Akams
+
+* GitHub: [@kamzzy](https://github.com/kamzzy)
+* Twitter:[@precious_akams](https://twitter.com/precious_akams)
+* LinkedIn:[Onyeagoziri Akams](https://www.linkedin.com/in/onyeagoziri-akams/)
+* AngelList:[Onyeagoziri Akams](https://angel.co/u/onyeagoziri-akams)
+***
+### 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+Show your support
+Give a ⭐️ if you like this project!
+***
+### Acknowledgments
+* Microverse
+ ***
+ ## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] It looks like you forgot to add MIT license to your project. Kindly add it before merging it 😊.